app/controllers/crate/version.js

```diff
@@ -26,7 +26,7 @@ export default Controller.extend({
   keywords: alias('crate.keywords'),
   categories: alias('crate.categories'),
   isOwner: computed('crate.owner_user', 'session.currentUser.id', function () {
-    return this.get('crate.owner_user').findBy('id', this.get('session.currentUser.id'));
+    return this.crate.owner_user.findBy('id', this.session.currentUser?.id);
```

Comment:
none of these are async relationships?

Reply:
hmm, good question, some of them actually are 🤔